insights/terraform/projects/bb2/services/common/main.tf

```diff
@@ -0,0 +1,26 @@
+locals {
+  tags = {
+    business    = "OEDA"
+    application = local.application
+    project     = local.project
+    Environment = local.env
+    stack       = "${local.application}-${local.project}-${local.env}"
+  }
+  application = "bfd-insights"
+  project     = "bb2"
+  env         = terraform.workspace
+  region      = "us-east-1"
+
+  # Shared lambda name/role (TODO: Split out by env)
```

Comment:
Is there a story or ticket we can reference here for this TODO?

Reply:
@billie-rose I created a story in our backlog for this to be groomed. TY!